packages/contracts-bedrock/src/L1/SimpleStorage.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+// Libraries
+import { Storage } from "src/libraries/Storage.sol";
+
+contract SimpleStorage is Initializable {
+    // Storage slots used to manage upgrades
+    bytes32 internal constant UPGRADED_1_3_0 = keccak256("UPGRADED_1_3_0");
```

Comment:
nit: we should make these preimage resistant

Reply:
we should yes.